dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/MainWindow.xaml

```diff
@@ -31,8 +31,7 @@
         xmlns:interop="clr-namespace:System.Windows.Interop;assembly=PresentationFramework"
         xmlns:local="clr-namespace:VisualLayerIntegration"
         mc:Ignorable="d"
-        Title="MainWindow" WindowState="Maximized"  
-        Loaded="Window_Loaded">
+        Title="MainWindow" WindowState="Maximized">
```

Comment:
Do you mean to force this to Maximized? People usually expect things to open the way they last opened.